android/src/main/java/com/tailscale/ipn/util/TSLog.kt

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+package com.tailscale.ipn.util
+
+import android.util.Log
+import libtailscale.Libtailscale
+
+object TSLog {
```

Comment:
Let's document why this exists and when it should be used in place of the util.Log.